spec/features/dams_object_spec.rb

```diff
@@ -867,4 +867,70 @@
     expect(page).to have_selector('div[id="map-canvas"][data=\'{"type":"point","coords":"19.7667,-154.8"}\']')
     expect(page).to have_css('img.leaflet-tile-loaded[src="https://c.tiles.mapbox.com/v4/mapquest.streets-mb/4/0/6.png?access_token=pk.eyJ1IjoibWFwcXVlc3QiLCJhIjoiY2Q2N2RlMmNhY2NiZTRkMzlmZjJmZDk0NWU0ZGJlNTMifQ.mPRiEubbajc6a5y9ISgydg"]')
   end
-end
\ No newline at end of file
+end
+
+describe "User wants to view a metadata-only view object " do
+  before(:all) do
+    ns = Rails.configuration.id_namespace
+    @note = DamsNote.create type: "local attribution", value: "Digital Library Development Program, UC San Diego, La Jolla, 92093-0175"
+    @localDisplay = DamsOtherRight.create permissionType: "localDisplay"
+    @metadataDisplay = DamsOtherRight.create permissionType: "metadataDisplay"
+    @metadataOnlyCollection = DamsProvenanceCollection.create titleValue: "Test UCSD IP only Collection with metadata-only visibility", visibility: "local"    
+    @localOnlyCollection = DamsProvenanceCollection.create titleValue: "Test UCSD IP only Collection with localDisplay visibility", visibility: "local"    
+    @collection = DamsProvenanceCollection.create titleValue: "Test UCSD IP only Collection with no localDisplay or metadata-only visibility", visibility: "local"    
+    @metadataOnlyObj = DamsObject.create titleValue: 'Test Object with metadataOnly Display', provenanceCollectionURI: @metadataOnlyCollection.pid, otherRightsURI: @metadataDisplay.pid, note_attributes: [{ id: RDF::URI.new("#{ns}#{@note.pid}") }], copyright_attributes: [{status: 'Public domain'}]
+    @localObj = DamsObject.create titleValue: 'Test Object with localDisplay', provenanceCollectionURI: @localOnlyCollection.pid, otherRightsURI: @localDisplay.pid, note_attributes: [{ id: RDF::URI.new("#{ns}#{@note.pid}") }], copyright_attributes: [{status: 'Public domain'}]
+    @obj = DamsObject.create titleValue: 'Test Object with no localDisplay, no metadataOnlyDisplay', provenanceCollectionURI: @localOnlyCollection.pid, copyright_attributes: [{status: 'Public domain'}]
+    solr_index @note.pid
+    solr_index @localDisplay.pid
+    solr_index @metadataDisplay.pid
+    solr_index @metadataOnlyCollection.pid
+    solr_index @localOnlyCollection.pid
+    solr_index @collection.pid       
+    solr_index @metadataOnlyObj.pid
+    solr_index @localObj.pid
+    solr_index @obj.pid
+  end
+
+  after(:all) do
+    @note.delete
+    @localDisplay.delete
+    @metadataDisplay.delete
+    @metadataOnlyCollection.delete
+    @localOnlyCollection.delete
+    @collection.delete   
+    @metadataOnlyObj.delete
+    @localObj.delete
+    @obj.delete
+  end
+
+  scenario 'should see Restricted View access control information' do
```

Comment:
Could the scenario titles perhaps refer to the kind of user and what they can or can't see? It might help if tests fail in the future to see what kind of access control/user types aren't seeing the right things.